### PR TITLE
modify a little

### PR DESCRIPTION
**Pull request by sunnyheero**
I think the new one is better.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 # IBMYes
 
+iiiiiiiii
```
Comment (moonlist) [inline README.md:3]: I belive you're right.

**Review by moonlist — COMMENTED** [review]
If you can fix a new problem, that'll be fine.

File: README.md (hunk 1 above)
Comment (sunnyheero) [reply README.md:3]: OK, I'll do it if I have time

**Comment by sunnyheero** [comment]
> If you can fix a new problem, that'll be fine.

ok,ok

**Pull request merged**